prov/efa/src/efa_base_ep.h

```diff
@@ -15,6 +15,9 @@
 #define EFA_QP_DEFAULT_SERVICE_LEVEL 0
 #define EFA_QP_LOW_LATENCY_SERVICE_LEVEL 8
 
+#define efa_rx_flags(efa_base_ep) ((efa_base_ep)->util_ep.rx_op_flags)
```

Comment:
nit: I don't see these macros actually simplifies code... maybe just use the original code it represents

Reply:
I was trying to be consistent with efa_rdm_rx_flags, efa_rdm_tx_flags, and smr_ep_rx_flags, smr_ep_tx_flags